### PR TITLE
Add grpc and grpc-java 1.69.0

### PR DESCRIPTION
**Pull request by mering**

### CODE REVIEW

**Comment by mering** [comment]
@bazel-io skip_check unstable_url

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (grpc-java, grpc) have been updated in this PR. Please review the changes.

File: modules/grpc/1.69.0/source.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "integrity": "sha256-OQy89POi9FtAhvbzPRSDSHyFeteql1ME09Ez54uJq7I=",
+    "strip_prefix": "grpc-1.69.0-pre1",
+    "url": "https://github.com/grpc/grpc/archive/refs/tags/v1.69.0-pre1.tar.gz",
```
Comment (mering) [inline modules/grpc/1.69.0/source.json:4]: Reminder: wait for stable release.

Comment (mering) [reply modules/grpc/1.69.0/source.json]: Done.

**Comment by mering** [comment]
gRPC 1.69 has been released. This PR is now ready for review.

**Review by fmeum — APPROVED** [review]

**Pull request merged**